jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/ImporterProcessor.java

```diff
@@ -72,15 +73,26 @@ public void processAndBackupFile(Path filePath, Path backupDir) {
       // ODE-559 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
       // ODE-559
+      boolean success = true;
       try (InputStream inputStream = new FileInputStream(filePath.toFile())) {
          BufferedInputStream bis = new BufferedInputStream(inputStream, odeProperties.getImportProcessorBufferSize());
          codecPublisher.publishFile(filePath, bis, fileType);
       } catch (Exception e) {
-         logger.error("Unable to open or process file: " + filePath, e);
+         success = false;
+         logger.error("Failed to open or process file: " + filePath, e);
+         EventLogger.logger.error("Failed to open or process file: " + filePath, e);  
       }
       
       try {
-         OdeFileUtils.backupFile(filePath, backupDir);
+         if (success) {
+            OdeFileUtils.backupFile(filePath, backupDir);
+            logger.info("File moved to backup: {}", backupDir);
+            EventLogger.logger.info("File moved to backup: {}", backupDir);  
+         } else {
+            OdeFileUtils.moveFile(filePath, failureDir);
+            logger.info("File moved to failure directory: {}", failureDir);  
```

Comment:
Since the same message is used in two places, it's better to define a String local variable. Use eclipse's `Refactor>Extract Local Variable` to do it for you.